_includes/samples/APIRequests/APIConsumer.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "@chainlink/contracts/src/v0.6/ChainlinkClient.sol";
+
+contract APIConsumer is ChainlinkClient {
```

Comment:
Update this example from https://github.com/smartcontractkit/documentation/pull/313/files and I think we are good to merge this PR.

Reply:
Also rebase to include #317

Reply:
Included changes from #313 and #317